src/plugins/plugin.tooltip.js

```diff
@@ -1001,4 +1002,49 @@ class Tooltip extends Element {
  */
 Tooltip.positioners = positioners;
 
-export default Tooltip;
+export default {
+	id: 'tooltip',
+	_element: Tooltip,
+	positioners,
+
+	afterInit: function(chart) {
+		const tooltipOpts = chart.options.tooltips;
+
+		if (tooltipOpts) {
```

Comment:
Just noticed this - is this safe? What if there are no `tooltipOpts` on first render, but then a chart update adds them. I think that might cause `chart.tooltip` to be `undefined`, which would mean the tooltip never gets turned on

Reply:
I'm not sure I can see an use case where this would happen. Fine this way by me.